extensions/wikia/CuratedContent/CuratedContentController.class.php

```diff
@@ -66,7 +66,7 @@ public function getPage() {
 		//This will always return json
 		$this->response->setFormat( 'json' );
 
-		$this->cacheResponseFor( 7, self::DAYS );
+		$this->cacheResponseFor( 24, self::HOURS );
```

Comment:
Why can't we call `setCacheValidity` here with one of the constant values taken from `WikiaResponse`?